ossec-server/usr/share/doc/ossec-server/changelog.Debian

```diff
@@ -1,3 +1,10 @@
+ossec-server (2.8.2) unstable; urgency=low
+
+  [ SecureDrop Team ]
```

Comment:
Do we need this annotation? Isn't the same information already present in the line below that starts with `--`? And isn't this true for every instance of this annotation?